privacyidea/lib/policy.py

```diff
@@ -2117,6 +2118,17 @@ def get_static_policy_definitions(scope=None):
                 'type': 'int',
                 'desc': _('Limit the number of allowed tokens in a realm.'),
                 'group': GROUP.TOKEN},
+            ACTION.REQUIRE_DESCRIPTION: {
+                'type': 'str',
+                'desc': _('During the rollout process, this policy makes the '
+                          'description required for all selected token-types.'),
+                'group': GROUP.ENROLLMENT,
+                'multiple': True,
+                'value': ['4eyes', 'certificate', 'email', 'hotp', 'indexedsecret',
```

Comment:
You can just use [`get_token_types()`](https://github.com/privacyidea/privacyidea/blob/49f64d01b2bffeacabeac721a405d46d9a62725d/privacyidea/lib/config.py#L481) here

Reply:
fixed in 79b20dd1280e6525c00936c96866e72094bde586.

Reply:
I accidentally wrapped get_token_types() in square brackets, which is why it didn't work for me without list().